control-plane/agents/src/common/errors.rs

```diff
@@ -376,6 +376,19 @@ pub enum SvcError {
         cluster_capacity_limit: u64,
         excess: u64,
     },
+    #[snafu(display("All replicas are not healthy for volume '{}'", id))]
```

Comment:
```suggestion
    #[snafu(display("All replicas are unhealthy for volume '{id}'"))]
```

Reply:
Don't we have similar error already?

Reply:
We have one error code which tells all the child are not healthy. This is the scenario, where some of the replicas are not healthy. Change the name of the error code and description. PTAL.